include/psen_scan/udp_client.h

```diff
@@ -179,13 +179,13 @@ inline void UdpClientImpl::sendCompleteHandler(const boost::system::error_code&
               << "Error message: " << error.message() << std::endl;
   }
   // LCOV_EXCL_STOP
-  std::cout << "Data successfully send." << std::endl;
+  std::cout << "Data successfully sent." << std::endl;
 }
 
-inline void UdpClientImpl::write(std::shared_ptr<char> data, const std::size_t& number_of_bytes)
+inline void UdpClientImpl::write(const std::vector<char>& data)
 {
-  io_service_.post([this, data, number_of_bytes]() {
-    socket_.async_send(boost::asio::buffer(data.get(), number_of_bytes),
+  io_service_.post([this, data]() {
+    socket_.async_send(boost::asio::buffer(&data.front(), data.size()),
```

Comment:
```suggestion
    socket_.async_send(boost::asio::buffer(&data.data(), data.size()),
```
https://stackoverflow.com/a/10740266/1493204

Reply:
Done.